app/grandchallenge/algorithms/models.py

```diff
@@ -419,6 +431,21 @@ def create_result(self, *, result: dict):
             )
             self.outputs.add(output_civ)
 
+    @property
+    def rendered_result_text(self):
+        interface = ComponentInterface.objects.get(slug="results-json-file")
+        try:
+            output = self.outputs.get(interface=interface)
+            if not self.algorithm_image.algorithm.result_template:
```

Comment:
We might need to add `"algorithm_image__algorithm"` to the `select_related` in the `AlgorithmJobsList` view to avoid an n+1 queries. If you turn on the debug toolbar are the number of SQL queries stable as the number of successful results with an algorithm template increase?

Reply:
Yes, the number of queries remains at 11, regardless of number of results, or whether I have a template defined.